docs/docs/emotion.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Emotion
+---
+
+In this guide, we'll walk through setting up a site with the CSS-in-JS library [Emotion](https://emotion.sh).
+
+Emotion is a performant and flexible CSS-in-JS library. Building on many other CSS-in-JS libraries, it allows you to style apps quickly with string or object styles. It has predictable composition to avoid specificity issues with CSS. With source maps and labels, Emotion has a great developer experience and great performance with heavy caching in production.
```

Comment:
This is good, but I would also recommend adding that it supports things like extracting critical styles _and_ rendering to CSS files, both of which using gatsby-plugin-emotion enable!

Reply:
Sure, I have added references to the suggested features.